general_itests/steps/local_run_steps.py

```diff
@@ -76,3 +76,15 @@ def local_run_on_chronos_job(context):
                          "--build "
                          "--cmd '/bin/sh -c \"sleep 2s && exit 42\"'")
         context.local_run_return_code, context.local_run_output = _run(command=local_run_cmd, timeout=30)
+
+
+@when(u'we run paasta local-run in non-interactive mode on a chronos job with cmd set to \'echo hello && sleep 5\'')
```

Comment:
I thought about it. My rationale was that one test gets the command as a parameter from the command line and the other reads it from the yaml file, so we end up testing two separate things.